src/components/ArchivePage/EventInfoItem.js

```diff
@@ -50,28 +53,54 @@ const useStyles = makeStyles(theme => ({
 	}
 }));
 
-function EventInfoItem({ name, mainLink, tags, directors, workshops, tagHighlight }) {
+function EventInfoItem({ name, mainLink, tags, directors, workshops, tagHighlight, slug }) {
 	const classes = useStyles();
-	const [isExpanded, setExpanded] = useState(true);
-	return <Accordion classes={{ root: classes.paperRoot }}
-		onChange={() => setExpanded(e => !e)}>
+	const [isExpanded, setExpanded] = useState(false);
+	console.log(name)
+	console.log(`window: `, window.location.hash)
```

Comment:
looks like this line is causing our build to fail. Try to remove it and see if running `yarn build` works.